VKProject/Src/Constants/ParameterNames.cs

```diff
@@ -0,0 +1,11 @@
+namespace VKProject.Constants;
+
+public class ParameterNames
+{
+    public const string OwnerId = "owner_id=";
+    public const string Attachments = "attachments=";
+    public const string PostId = "post_id=";
+    public const string Message = "message=";
+    public const string Version = "v=";
+    public const string Token = "access_token=";
```

Comment:
there's everywhere '=' in string lines, can you optimize it?

Reply:
discussed